ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,160 @@
+
+name: Monorepo labels
+description: Import on 2023-05-09
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: B
+    name: Breaking changes 
+    description: Upgrade dependencies, relevant for downstream Parachains/Parathreads.
+  - code: C
+    name: Complexity 
+    description: Complexity of the issues, also used for contribution guidelines.
+  - code: D
+    name: Auditing 
+    description: Required when there were changes in the paths [CI/CD INPUT NEW FILE PATHS]. Also used to indicate the need for a documentation audit of DevRel.
+  - code: I
+    name: Issues 
+    description: Categorisation of issues
+  - code: T
+    name: Topics
+    description: PR/Issue is related to this topic
+
+labels:
+  - name: A0-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps team is notified via matrix.
+    color: c7def8
+  - name: A1-insubstantial
+    description: Pull request requires no code review (e.g., a sub-repository hash update).
+    color: C6C7F8
+  - name: A2-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A3-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: B0-runtime_migration
+    description: PR introduces code that might require downstream chains to run a runtime upgrade.
+    color: c2e0c6
+  - name: B1-database_migration
+    description: PR introduces code that does a one-way migration of the database.
+    color: f4f5f9
+  - name: B2-node_first_update
+    description: This is a runtime change that will require all nodes to be update BEFORE the runtime upgrade.
+    color: e8d7d7
+  - name: B3-add_new_host_function_to_client
+    description: PR adds a new host function to the client which is not yet used by the runtime.
+    color: f9f2f2
+  - name: B4-use_previously_unused_host_function_in_runtime
+    description: PR modifies the runtime to use a new host function; this raises the minimum client node version requirement for the next runtime upgrade!
+    color: f9f2f2
+  - name: B5-unused_host_function_breaking_change
+    description: PR introduces a breaking change to the behavior of a host function which is not used by any runtime.
+    color: f9f2f2
+  - name: B6-remove_unused_host_function
+    description: PR removes a host function not used by any runtime.
+    color: f9f2f2
```

Comment:
All of them are basically release notes related.

Reply:
We have only ever used the `runtime_migration` label to create awareness. I am happy to remove those labels as long as we have an analog way to include the migration notice in the release notes.

Reply:
we're aiming for proper changelogs where you can add any kind of important information focused on particular audiences as well: https://github.com/orgs/paritytech/projects/90?pane=issue&itemId=27942554